deployment/selinux.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+if [ $# -lt 1 ]; then
+	_var=/var
+else
+	_var=$1
+fi
+
+if /usr/sbin/selinuxenabled ; then
+    semanage fcontext -a -t httpd_sys_content_t "${_var}/lib/crane(/.*)?"
```

Comment:
This line is called by both usages in the spec file, but notice that the ones in the spec file have different options. One has `-a` and another has `-d`.